com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/test/util/MockResource.java

```diff
@@ -44,14 +44,6 @@ public MockResource(MockFileSystem fileSystem, String directory) {
 
     @Override
     public void setContent(byte[] content) {
-        if (!isOutput()) {
```

Comment:
This was interesting. I have no idea why you'd ever want a mock implementation behave differently from the thing it is supposed to mock? It is perfectly ok to use setContent() on the DefaultResource implementation of IResource.